src/client/pages/Events/EventCard.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import Card from '@material-ui/core/Card';
+import EventDate from './EventDate.jsx';
+import EventContent from './EventContent.jsx';
+
+const styles = {
```

Comment:
You may want to look into what this does...
```suggestion
 const styles = (theme) => ({ /* styles */ })
```

Reply:
I see. I will change it